packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -305,7 +313,7 @@ export class Coachmark extends BaseComponent<ICoachmarkProps, ICoachmarkState> {
           this.forceUpdate();
         }
 
-        this._events.on(document, 'keydown', this._onKeyDown, true);
+        this._addListeners();
```

Comment:
You also need to have this in componentdidupdate, since preventDismissOnLostFocus is a prop you'll need to add/remove them when and if those props are updated.